Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -2855,35 +2956,48 @@ private Boolean isPointInside(Double lat, Double lng) {
     @JavascriptInterface
     public void locateOnMap(boolean goToCurrentLocation, final String lat, final String lon) {
         try {
```

Comment:
This try/catch block is not used this can be removed
Just add a null check for activity